terraform/workspace-variables/qa.tfvars

```diff
@@ -1,6 +1,4 @@
 # PaaS
-paas_user                           = ""
-paas_password                       = ""
 paas_sso_passcode                   = ""
 paas_app_environment                = "qa"
 paas_app_docker_image               = "dfetwdregister/register-trainee-teacher-data:2362f39"
```

Comment:
This is not very useful if we hardcode the docker image tag. We discussed some possible solutions:
1. * Move the deploy job to the build workflow
    * Add a conditional to deploy only if the branch is master
    * Pass tag of newly built docker image to terraform
1. * Do not trigger the deploy workflow from push
    * Trigger deploy workflow only via `workflow_dispatch`
    * Add [triggering](https://github.com/marketplace/actions/workflow-dispatch) from build workflow **if** branch is master
1. Other option?

@misaka any preference?

Reply:
having 2 jobs (`build` & `deploy`) sounds simple for now,
and having a condition on the `deploy` job to only be run if pushed to `master`

Reply:
i have also gone with this....

Reply:
I agree with having a separate workflow for deploy. Is it a problem if we merge this for now and do that change as a new piece of work?

Reply:
This is not a problem; it is the same workflow for now but we can address in future dates...?